packages/cli/src/commands/container/release.ts

```diff
@@ -0,0 +1,112 @@
+import color from '@heroku-cli/color'
+import {Command, flags} from '@heroku-cli/command'
+import {ux} from '@oclif/core'
+import * as Heroku from '@heroku-cli/schema'
+import {debug} from '../../lib/container/debug'
+import {streamer} from '../../lib/container/streamer'
+
+type ImageResponse = {
+  schemaVersion: number,
+  history: [{
+    v1Compatibility: string,
+    }
+  ],
+  config: {
+    digest: string
+  }
+}
+
+export default class ContainerRelease extends Command {
+  static topic = 'container'
+  static description = 'Releases previously pushed Docker images to your Heroku app'
+  static usage = 'heroku container:release'
+  static example = `
+  ${color.cmd('heroku container:release web')}        # Releases the previously pushed web process type
+  ${color.cmd('heroku container:release web worker')} # Releases the previously pushed web and worker process types`
+
+  static strict = false
+
+  static flags = {
+    app: flags.app({required: true}),
+    verbose: flags.boolean({char: 'v'}),
+  }
+
+  async run() {
+    const {flags, argv} = await this.parse(ContainerRelease)
+    const {app, verbose} = flags
+
+    if (argv.length === 0) {
+      this.error(`Error: Requires one or more process types\n ${ContainerRelease.example}`)
+    }
+
+    if (verbose) {
+      debug.enabled = true
+    }
+
+    await this.heroku.get(`/apps/${app}`)
```

Comment:
Is this intentional? Possibly to throw at this point of the app does does not exist?

Reply:
Yes, that's done to verify that the user has access to the app where the images will be released before attempting anything else. If the user has lost access to the app, the release step will fail, but checking this first it avoids to pull the images when the next steps will fail.